website/content/docs/commands/hcl2_upgrade.mdx

```diff
@@ -98,9 +98,11 @@ working on improving this part of the transformer.
 
 ## Options
 
-- `-output-file` - File where to put the hcl2 generated config. Defaults to
-  JSON_TEMPLATE.pkr.hcl
-- `-with-annotations` - Adds helper annotations with information about the generated HCL2 blocks.
+- `-output-file` - Filename of the hcl2 generated config. Defaults to
+  JSON_TEMPLATE.pkr.hcl; for example, if your template is called
+  "packerparty.json", the default output-file is "packerparty.json.pkr.hcl".
```

Comment:
output-file -> output-template

Reply:
```suggestion
  "packerparty.json", the default output-template is "packerparty.json.pkr.hcl".
```

Reply:
It's also a file ! I'd think this is okay